packages/framework/fluid-framework/api-report/fluid-framework.public.api.md

```diff
@@ -26,9 +31,17 @@ import { PropertiesManager } from '@fluidframework/merge-tree/internal';
 import { PropertySet } from '@fluidframework/merge-tree/internal';
 import { ReferencePosition } from '@fluidframework/merge-tree/internal';
 import { ReferenceType } from '@fluidframework/merge-tree/internal';
+import { SchemaLibrary as SchemaLibrary_2 } from '../feature-libraries/schemaBuilderBase.js';
```

Comment:
Have we checked the rollups generated for this package? This import statement is invalid. This is a known API-Extractor bug (https://github.com/microsoft/rushstack/issues/4507), but also often appears as an indicator of other things going wrong.